conf/config.py

```diff
@@ -220,6 +220,9 @@ class DevConfiguration(BaseConfiguration):
 
     LIGHTBLUE_VERIFY_SSL = False
 
+    # TODO add to PROD and DEV configs
+    PYXIS_SERVER_URL = 'https://pyxis.stage.engineering.redhat.com/'
```

Comment:
I would not set this value upstream since it's an internal URL. This can be set in the deployment configuration code.